index.html

```diff
@@ -1,12 +1,131 @@
-<!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+  <header class="page-header">
+    <nav class="navigation">
+      <img src="logo.svg" alt="Город Седона." width="40" height="30">
+      <ul class="navigation-list">
+        <li class="navigaton-item">
+          <a class="navigation-link" href="#">Главная</a>
+        </li>
+        <li class="navigaton-item">
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="navigation-user">
+        <li class="navigaton-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Поиск.</span>
+          </a>
+        </li>
+        <li class="navigaton-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Избранное.</span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Хочу сюда.</span>
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="main-invitation">
+
+    <!--Вставлю картинку в 3-м дз -->
+    <section class="preview">
+      <h1>Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</h1>
+      <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+      <ol class="bloks">
+        <li class="block1">
+          <h2>Настоящий городок</h2>
+          <p>Седона — не аттракцион<br> для туристов, там течёт<br>своя жизнь</p>
+          <img src="" alt="Фасад домов.">
+        </li>
+        <ul class="miniblock">
+          <li class="mini1">
+            <img src="" alt="Дом.">
+            <h3>Жильё</h3>
+            <p>Рекомендуем пожить<br>в настоящем мотеле,<br>всё как в кино!</p>
+          </li>
+          <li class="mini2">
+            <img src="" alt="Бургер.">
+            <h3>Еда</h3>
+            <p>Всегда заказывайте<br>топовый фирменный бургер,<br> вы не разочаруетесь!</p>
+          </li>
+          <li class="mini3">
+            <img scr="" alt="Перевязанная коробка">
```

Comment:
```suggestion
            <img scr="" alt="Перевязанная коробка.">
```

всегда ставь точки в альтах

Reply:
поправь дальше по коду